supplier_app/__init__.py

```diff
@@ -4,6 +4,14 @@
 
 DBTuple = namedtuple('DBTuple', 'value verbose_name')
 
+EMAIL_SUCCESS_MESSAGE = "Your invitation was sent successfully"
+EMAIL_ERROR_MESSAGE = "There was an error sending your email"
+
+TAXPAYER_CREATION_SUCCESS_MESSAGE = "Your taxpayer was created successfully"
+TAXPAYER_CREATION_ERROR_MESSAGE = "There was an error on creating your taxpayer please try again"
+TAXPAYER_FORM_INVALID_MESSAGE = "Your form is invalid"
+COMPANY_ERROR_MESSAGE = "You don't have a company yet please contact with us 'mail@mail.com'"
```

Comment:
Are we aware of `i18n` for this?

Reply:
For now they are only available in english, I think when we have to add another language we have to add new special messages for that language.
Do you think I have to make a function(where it receive a language and then he return me the message for that language) now or I have to consider for the future?